### ISSUE TITLE
Fixed drawer has no shadow on Firefox

### ISSUE THREAD

**Issue by mewa** [google/material-design-lite#4992]
Fixed drawer has no shadow/elevation on Firefox. It is present on Safari and Chrome. 
You can preview this behavior on getmdl.io in layouts section. 
I was actually surprised, because the same can be observed on the main material design site (navigation drawers > permanent section).

**Comment by sgomes** [comment]
Thanks for the bug report, @mewa! I'm having some trouble reproducing your bug, though, as things seem to be working fine on my end:

![image](https://cloud.githubusercontent.com/assets/409615/21544391/518be886-cdc6-11e6-84cd-fbdcae9e8fe6.png)

Could you provide some more details on your setup, namely Firefox and OS versions?

**Comment by mewa** [comment]
@sgomes It's Firefox 50.1.0 on Ubuntu 16.04 LTS. 

The weird thing however is, that while I could normally see your screenshot on a Windows Chrome, i. e., with proper elevation, the same screenshot viewed on Ubuntu didn't have it. 

As seen on getmdl.io on this setup: 
![screenshot from 2016-12-29 23-07-26](https://cloud.githubusercontent.com/assets/7347194/21555853/300b30bc-ce1c-11e6-8aea-07e0653d20bd.png)

This led me to thinking it could be GPU driver-related. I'm going to update them and check out the results.

**Comment by sgomes** [comment]
So the issue is present in both Chrome and Firefox, in your setup? That does indeed sound like some sort of graphical glitch on your end, albeit an odd one. Let me know if you figure anything out!

**Comment by sgomes** [comment]
Actually, now that I think about it, it could be a simple screen alpha issue that's reducing contrast so much that the shadow becomes invisible. Could you share a screenshot without the orange overlay, so that I can take a closer look at the pixels?

**Comment by mewa** [comment]
@sgomes I was trying to fix it at hand and have replaced my GPU drivers several times in the process. 
I failed to install something other than the basic GPU drivers (AMD eh...) and had to replace the junk from failed attempts entirely, but it seems the "basic" driver got updated and while I was cleaning up I must've downloaded a newer version this time. 
Summing it up, yep - definitely GPU driver-related. It's a shame I didn't write down driver version before starting... but at least it's back to normal now.
